docs/providers/aws/guide/quick-start.md

```diff
@@ -25,6 +25,8 @@ Create a new service using the Node.js template, specifying a unique name and an
 $ serverless create --template aws-nodejs --path my-service
 # Change into the newly created directory
 $ cd my-service
+# Generate the package.json file
```

Comment:
Could we remove the whole `npm` related processes since the `aws-nodejs` template has no `package.json` at all.

This would make the quickstart even quicker.